src/test/java/com/microsoft/sqlserver/jdbc/databasemetadata/DatabaseMetaDataForeignKeyTest.java

```diff
@@ -84,6 +84,34 @@ public static void setupVariation() throws SQLException {
                     + "(c41) ON DELETE set null ON UPDATE cascade," + " c15 int FOREIGN KEY REFERENCES "
                     + AbstractSQLGenerator.escapeIdentifier(table5) + "(c51) ON DELETE set default ON UPDATE no action,"
                     + ")");
+
+            TestUtils.dropSchemaIfExists(anotherSchema, stmt);
```

Comment:
`dropSchemaIfExists()` will fail if the schema contains any objects, for example a table. If you make `anotherSchema` a random name, you won't need to drop the schema here, it will need to be dropped only after the test is done.